src/screens/Debug/DebugIndex.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button } from 'react-native'
+import { Box } from '@components'
+
+export const DebugIndex = ({ navigation }) => {
```

Comment:
Defining React components like this causes them to be labelled `Anonymous` in React Dev tools. Not sure if that's relevant in our case but I think it would still be a good idea to define these properly.

```jsx
const DebugIndex = () => ()
export { DebugIndex }
```

Reply:
It does?

Normally `export const foo = ` is fine and `export default ...` is bad.

Reply:
Because the `Anonymous` part happens because when a default is exported it doesn't have a name searchable in the AST and is so anonymous.

Reply:
Right okay cool. I wasn't sure so if it works as is then that's fine.